### PR TITLE
Create developer onboarding documentation

### PR DESCRIPTION
**Pull request by tekenny**
PR for issue #121 

There are 3 TBDs so let me know if you have any details for them or if there are any other suggestions for improvements.

Thanks!

### CODE REVIEW

File: .github/DEVELOPERS.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-to be able to contribute to the project.
+The purpose of this document is to document the steps for a new developer of this project to get onboarded
+to be able to contribute.
 
 1 - The source code for this application is written in the Pascal programming language.
-Only Windows operating systems are currently supported as the target platform for the application to run.
+Only Windows operating systems are currently supported as the target platform for the application to operate on.
 
 1 - Fork the repo to your own GitHub account.
+git clone https://github.com/<yourAccountName>/MourseRunner.git
```
Comment (w7sst) [inline .github/DEVELOPERS.md:8]: <yourAccountName>/MourseRunner.git should be w7sst/MorseRunner.git

**Review by w7sst — APPROVED** [review]

**Comment by w7sst** [comment]
@tekenny 
Hi Tom, I do not have any input on the TBD's at this time. We can circle back on this later when I get some time. Thanks, Mike

File: .github/DEVELOPERS.md (hunk 1 above)
Comment (tekenny) [reply .github/DEVELOPERS.md:8]: I need some time to work out if what you stated would work do a fork and clone in one command.
I've realized that I neglected to state how to do a fork. It seemed like a no brainer to me, doing such on the GitHub GUI so many times previously on other projects.
So I'll be making a change, please don't merge this yet.

**Comment by tekenny** [comment]
@w7sst Agreed, I'll leave the TBDs in the document at this time. Better to have most of the information documented even with a little tech debt :-) rather than incomplete info.

File: .github/DEVELOPERS.md (hunk 1 above)
Comment (w7sst) [reply .github/DEVELOPERS.md:8]: Ahhh. Your instructions had assumed that you had created a fork (like you did under your user account). My change assumed you were cloning directly to your home machine. Yes, I think most people will create a fork.

Let me know when you want me to resume the review.

**Comment by tekenny** [comment]
Just an update - I'm still researching good resources and proper information for this PR so not done yet...

**Comment by tekenny** [comment]
I've updated DEVELOPERS.md and think it is ready to be reviewed again and possibly merged.

**Review by w7sst — APPROVED** [review]
Looks great. I'll circle back at some point and fill in some of the TBDs.

**Pull request merged**